config/locales/fr.yml

```diff
@@ -57,6 +61,8 @@ fr:
     reading_list: Liste de lecture
     reply: Répondre
     report_abuse: Signaler un abus
+    save: Enregistrer
+    saved: Enregistré
```

Comment:
The two lines not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L49-L50
(look a bit too long compared to English button width; we may hear native speakers if there are any remedies)